pom.xml

```diff
@@ -54,6 +49,12 @@
             <artifactId>spring-security-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.1.0</version>
```

Comment:
seems like this was missing? Not sure if tests run in Eclipse without this dep or not, but it will fail from the command line without it

Reply:
Yeah, I think that Eclipse provides this, but you are right, this will be good for those running outside of eclipse

Reply:
Thanks Brian!  I am not merging this PR because I wanted to manually merge it so I understood everything fully.  I also kept my IT* naming as the failsafe plugin understands that variation by default as well.  I should send a notice that I am done with edits soon.

Reply:
Sweet! i didn't know that